edkahara/iReporter
# Ft create v2 get specific report endpoint #162423208

__What does this PR do__
Create a get specific report endpoint for version 2 of iReporter.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the getting of a specific report. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-get-specific-report-endpoint-#162423208 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetest.

__Relevant pivotal tracker stories.__
#162423208
